src/main/java/com/gofurther/emissora/entity/Reservation.java

```diff
@@ -0,0 +1,51 @@
+package com.gofurther.emissora.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.Date;
+
+@Entity
+public class Reservation extends User {
+    @Id
+    @GeneratedValue
+    @Column(name="id_reservation")
+    private int idReservation;
+    @Column(name="date_start")
+    private Date start;
+    @Column(name="date_and")
+    private Date and;
+    @Column(name="fk_actor")
```

Comment:
```suggestion
    @ManyToOne
    @JoinColumn(name="id_actor)
```

Reply:
Quando for criar a entidade actor por exemplo
no atributo Id coloca @Column(name="id_actor)